project-reports/2024/2024-annual-Hyperledger-Iroha.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: default
+title: Hyperledger Iroha 2024 Annual Review 
+parent: 2024
+grand_parent: Project Updates
+---
+
+# Project Health
+
+Our LFX Insights page is available with an [old](https://insights-v2.lfx.linuxfoundation.org/iroha/trends?selectedDateFilterType=DATERANGE&selectedDateRangeKey=1Y) and [new](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=iroha&bestPractice=false&repository=all&dateFilters=Last%2012%20Months&dateRange=2023-02-02%20to%202024-02-01&compare=PP&granularity=month&hideBots=true) interface. It must be noted that it doesn't reflect the whole picture, as we have several SDKs ([`iroha-java`](https://github.com/hyperledger/iroha-java), [`iroha-javascript`](https://github.com/hyperledger/iroha-java)) and the documentation repositiory, [`iroha-2-docs`](https://github.com/hyperledger/iroha-2-docs).
```

Comment:
It is lots of fun trying to find the best LFX Insights links to use!  After looking at the linked metrics in the report that used the two different LFX Insight versions, I generated my own using the latest LFX Insights version, one for all of 2022, and one for all of 2023.  Thus -- I got these links:

- https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=iroha&bestPractice=false&repository=all&dateFilters=2022-01-01%20to%202022-12-31&dateRange=2022-01-01%20to%202022-12-31&compare=PP&granularity=month&hideBots=true

- https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=iroha&bestPractice=false&repository=all&dateFilters=2023-01-01%20to%202023-12-31&dateRange=2023-01-01%20to%202023-12-31&compare=PP&granularity=month&hideBots=true

Two notes on the comment about the whole picture.  There is a typo in the `iroha-javascript` target. Regardless, using the links I used, the "repository" drop down (default `All`) does show the other repos are included in the metrics.